docs/writing/posts/learning.md

```diff
@@ -0,0 +1,153 @@
+
+#### Process vs Outcome
+
+I was holding the work a little too precious, and I wasn't actually putting in any volume.
+
+What did I learn? I basically learned that we often set outcome-based goals instead of process-based goals, and even when we do set process-based goals, we are definitely not using the right measurement. No wonder I wasn't doing enough to actually achieve the outcomes that I wanted. Moreover, I was also not measuring the right thing, usually the units and order of manitude were off and figuring out the right metric bags of clay was a huge part of the process.
+
+### Weightlifting
+
+By the time I started weightlifting, I kind of accepted already that I had to focus on process based goals. When I learned about progressive overload, it made sense that if my metric was just `weight * reps * sets` all I had to do was to increase that number every week. I could increase, weight, reps, or sets, every week, and I would be making progress.
+
+#### Periodize your training
+
+I learned that not only do strength athletes do progressive overload, they also periodize their training. They have a hypertrophy phase, a strength phase, and a peaking phase. Not only that, they take deload weeks! What I got from my time weightlifting is that for anything that I'm trying to learn, not only should I think about volume instead of outcomes, I should also think about how I can do progressive overload to increase volume and periodize the training and apply rest as a feature of the training program. Many professional also both on season and off-season where they can focus on different aspects of their training like skills development vs conditioning.
+
+Its also just more fun to set learning goals and peak over some multimonth period. It makes the expectation of progress more realistic and it's easier to stay motivated.
+
+### Jiu Jitsu
+
+I think Jiu Jitsu is by far the most interesting thing I have ever done. There are so many lessons I've learned from practicing it, but I want to focus on the idea of choosing your teachers. There is a huge advantage in choosing to learn from active competitors. I believe that techniques and tips in combat sports, in particular, are very difficult bullshit. It's quite easy to watch somebody on YouTube telling you how to make $1 million, but you'll never be able to verify if they have actually achieved that themselves.
```

Comment:
What do you mean by "difficult bullshit"? Like "difficult to bullshit"? Or more like "difficult to discern if they're bullshit"? Or "difficult shit"?